### PR TITLE
AST-1860 - No spacing/margin or block to add between multiple paragraphs - theme v3.8.0

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description
We have seen version Astra v3.8.0, there is no space between paragraph blocks and add another block option. verified with Astra v37.10 and twenty twenty-two theme.

*Tickets:*
https://secure.helpscout.net/conversation/1876295749/184599?folderId=4660463

No margin between the paragraph blocks in the editor (https://d.pr/i/sh0F7e), but it displays fine on the fronted (https://d.pr/i/3gNytq).

### Types of changes

 Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
Added in the card!

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by nileshc-bsf — COMMENTED** [review]
Add the changelog

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+v3.8.1
+- Fix: Block editor - Margin is not set within Paragraph block.
```
Comment (premanshup) [inline changelog.txt:2]:
```suggestion
- Fix: Block editor - Margin is not set within Paragraph block.

```

File: inc/core/class-astra-wp-editor-css.php [hunk 2]
```diff
@@ -257,7 +257,6 @@ public static function get_css() {
 			),
 			'.editor-styles-wrapper .block-editor-block-list__layout.is-root-container > p, .editor-styles-wrapper .block-editor-block-list__layout.is-root-container .block-list-appender' => array(
 				'max-width' => 'var(--wp--custom--ast-wide-width-size)',
-				'margin'    => '0 auto',
```
Comment (imnavanath) [inline inc/core/class-astra-wp-editor-css.php:260]: Removing this CSS is not solution. This CSS is for making layout center align. Try different solution.

File: changelog.txt (hunk 1 above)
Comment (GauravKhupse) [reply changelog.txt:2]: changes done!

File: inc/core/class-astra-wp-editor-css.php (hunk 2 above)
Comment (GauravKhupse) [reply inc/core/class-astra-wp-editor-css.php:260]: I have added specific margin for paragraph block , confirm once

**Comment by imnavanath** [comment]
@GauravKhupse  I think we can close this PR, as this I included in block editor v2 PR, here is the commit for same - https://github.com/brainstormforce/astra/pull/4352/commits/9a82fcbd95afadd25565daa377f81a77ec62dd70

**Review by premanshup — APPROVED** [review]

**Comment by imnavanath** [comment]
Closing this PR as this is done in better way here - https://github.com/brainstormforce/astra/commit/9a82fcbd95afadd25565daa377f81a77ec62dd70 in [Block editor v2 update PR](https://github.com/brainstormforce/astra/pull/4352).

**Pull request closed without merging**